packages/server/marketplaces/chatflows/Web Search QnA.json

```diff
@@ -0,0 +1,570 @@
+{
```

Comment:
thanks for the template! I think this is very similar to other existing template in marketplace like WebPageQnA and ConversationalRetrievalQAChain and etc, so maybe we can skip having a template for this

Reply:
alright